### PR TITLE
Fixes#250. Update index.html

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #250 
Added link to map marker 
also added an hr at the bottom

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -896,7 +896,7 @@ <h1>How Can I Join Projects</h1>
 								<li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
 								<li><i class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
 								<li><span>Ms. DANG Hong Phuc</span></li>
-								<li><span>6 Eu Tong Sen Street #05-07 The Central, Singapore 059817</span></li>
+								<li><span><a href="https://www.google.co.in/maps/place/Clarke+Quay+Central/@1.2891461,103.8443159,17z/data=!4m8!1m2!2m1!1s6+Eu+Tong+Sen+Street+%2305-07+The+Central,+Singapore+059817!3m4!1s0x31da190a9bbec0d7:0x4d829ccbc19a9496!8m2!3d1.2889924!4d103.8467458?hl=en" style="color:#777777" id="mailAdd">6 Eu Tong Sen Street #05-07 The Central, Singapore 059817</a></span></li>
```
Comment (abishekvashok) [inline index.html]: Please use open street map link

Comment (arjunpandey2003) [reply index.html]: Sure wait

**Comment by arjunpandey2003** [comment]
@abishekvashok done

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/labs.fossasiapcd.org/.
deploy link

**Comment by arjunpandey2003** [comment]
id mailAdd is for the white line underneath

**Comment by arjunpandey2003** [comment]
Will keep the inline point in mind next time sorry!

**Comment by arjunpandey2003** [comment]
Sure @abishekvashok

**Comment by arjunpandey2003** [comment]
@abishekvashok done

**Comment by duttaditya18** [comment]
@arjunpandey2003 Awesome.

**Comment by arjunpandey2003** [comment]
@duttaditya18 thanks!

**Comment by abishekvashok** [comment]
We are ready to ship this.
Thank you @arjunpandey2003

**Comment by arjunpandey2003** [comment]
@abishekvashok :)

**Comment by arjunpandey2003** [comment]
@ParthS007 thanks i have also done changes on the other PR

**Comment by arjunpandey2003** [comment]
@ms10398 thanks!

**Pull request merged**